org.eclipse.winery.topologymodeler.ui/src/app/canvas/canvas.component.ts

```diff
@@ -1,38 +1,27 @@
-import {
-  Component,
-  DoCheck,
-  ElementRef,
-  HostListener,
-  Input,
-  KeyValueDiffers, OnDestroy,
-  OnInit,
-} from '@angular/core';
-import { JsPlumbService } from '../jsPlumbService';
-import { JsonService } from '../jsonService/json.service';
-import {TNodeTemplate, TRelationshipTemplate, TTopologyTemplate} from '../ttopology-template';
-import { LayoutDirective } from '../layout.directive';
+import {Component, ElementRef, HostListener, KeyValueDiffers, OnDestroy, OnInit,} from '@angular/core';
+import {JsPlumbService} from '../jsPlumbService';
+import {JsonService} from '../jsonService/json.service';
+import {TNodeTemplate, TRelationshipTemplate} from '../ttopology-template';
+import {LayoutDirective} from '../layout.directive';
 import {AppActions} from '../redux/actions/app.actions';
-import {NgRedux, select} from '@angular-redux/store';
+import {NgRedux} from '@angular-redux/store';
 import {IAppState} from '../redux/store/app.store';
-import {Observable} from 'rxjs/Observable';
+import {ButtonsStateModel} from '../models/buttonsState.model';
+import {TopologyRendererActions} from '../redux/actions/topologyRenderer.actions';
 
 @Component({
   selector: 'app-canvas',
```

Comment:
Currently TSLint complains about any other prefix than "app". You can try editing the tslint.json by replacing the occurrences of "app" with "winery"

"directive-selector": [true, "attribute", "app", "camelCase"],
"component-selector": [true, "element", "app", "kebab-case"],
I already set the prefix to "winery" in our angular-cli.json inside my pending pull request.